auto_py_to_exe/web/js/i18n.js

```diff
@@ -1031,7 +1066,8 @@ const translationMap = {
         ko: '결과물이 저장될 디렉토리. 디렉토리가 존재하지 않으면 자동 생성됩니다.',
         bg: 'Изходната папка ще бъде създадена, ако не съществува.',
       },
-      increaseRecursionLimit: {
+      increaseRecursionLimhe: '',
+        it: {
```

Comment:
Are you sure this runs? This is the same error that caused it not to run previously.

Reply:
no. this is old. I pushed another commit with fixed code. 
I have now pushed (again) fix for this. 
sorry for the repo confusion and code ping-pong.